modules/configuring-a-proxy-after-installation-cli.adoc

```diff
@@ -30,17 +30,27 @@ $ rosa edit cluster \
  --cluster $CLUSTER_NAME \
  --additional-trust-bundle-file <path_to_ca_bundle_file> \ <1> <2> <3>
  --http-proxy http://<username>:<password>@<ip>:<port> \ <1> <4>
- --https-proxy http(s)://<username>:<password>@<ip>:<port> <1> <4>
+ --https-proxy http(s)://<username>:<password>@<ip>:<port> \ <1> <4>
```

Comment:
If this is explicitly for HTTPS, does it need `(s)`? There's one for HTTP, so specifying HTTP for an HTTPS might not be valid?